Src/Plugins/Security/Utilities/AlibabaEcsRequestSigner.cs

```diff
@@ -39,6 +39,9 @@ public AlibabaEcsRequestSigner(DateTime? dateTime = null, string signatureNonce
         /// Sign the HTTP request using Alibaba Cloud access key and secret so it
         /// can be used to access Alibaba Cloud service REST API.
         /// </summary>
+        /// <param name="request">Unsigned httpRequest passed in that will be signed for Alibaba Cloud service REST API.</param>
```

Comment:
>httpRequest [](http://example.com/codeflow?start=43&length=11)

what is this term? it's not a variable name, it's not a type, it's not a valid english term. ? :)

Reply:
+1. I couldn't parse this.